vncdotool/client.py

```diff
@@ -124,7 +124,7 @@ class AuthenticationError(Exception):
 
 
 class VNCDoToolClient(rfb.RFBClient):
-    encoding = rfb.RAW_ENCODING
+    encoding = rfb.ZRLE_ENCODING
```

Comment:
I'm on the fence about changing the default.  I tend to agree with your logic but I'm not sure what general compatibility of servers is with ZRLE and worry about breaking existing users.  I see a couple of options I outline below and am open to others. 

The quickest path to merging this PR would be to keep the default as RAW.  You could then expose a command line option to control the selection of encoding in a follow up PR.  Finally, in a major version bump we can change the default from RAW to ZRLE.  

Another option is I hold this PR up until a major version bump is approaching and then add the CLI switch defaulting to ZRLE.

Reply:
I have reverted the change of the default encoding